tools/releasing/collect_license_files.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# This script extracts from all jars in the specified directory the NOTICE files and the
+# licenses folders. It then concatenates all NOTICE files and collects the contents of all
+# licenses folders in the specified output directory.
+#
+# This tool can be used to generate a rough skeleton for the binary NOTICE file. Be aware,
+# that it does not deduplicate contents.
+
+SRC=${1:-.}
```

Comment:
set fail on error flags and all that stuff?

Reply:
Good point. Will add it.